doc/README.msys2

```diff
@@ -65,7 +59,7 @@ using MSYS2 from http://msys2.github.io
 
 2.1.2) Arch-specific packages needed for building freeciv
 
-2.1.2.1) Arch-specific packages for building common parts
+2.1.2.1) Arch-specific packages for building common parts (i686 or x86_64, not both)
```

Comment:
Does it fail if you install both?

Reply:
You get issues and have to tell the make process what you want to build for. My scripts only install x86_64.

Reply:
I see. Are 32 bits PCs still in use though? Some major Linux distros dropped support…

Reply:
Fixed in third commit